Library/Formula/postgresql.rb

```diff
@@ -77,8 +82,16 @@ def install
   end
 
   def post_install
-    unless File.exist? "#{var}/postgres"
-      system "#{bin}/initdb", "#{var}/postgres"
+    system "pkill postgres || true"
+
+    if (var/"postgres").exist?
+      mv var/"postgres", var/"postgres-#{version_suffix}"
+    elsif (var/"postgres-#{version_suffix}").exist?
+      puts ""
```

Comment:
Cool, If that works I'll just axe those two lines. Pain about the ` and ` not being covered, is there a way to get them covered? Per Tim's comments earlier, we need to cover the situation where someone has ended up with both ` Postgres ` and ` Postgres-4.9 ` folders in {var}. Any strong feelings about what we should do in that situation?